docker/build-tools/Dockerfile

```diff
@@ -53,9 +54,15 @@ RUN GO111MODULE=on go get golang.org/x/tools/cmd/goimports@${GOIMPORTS_VERSION}
 RUN GO111MODULE=on go get github.com/golangci/golangci-lint/cmd/golangci-lint@${GOLANGCI_LINT_VERSION}
 RUN go get github.com/jteeuwen/go-bindata/go-bindata
 
-# Install latest versions of custom Istio tools
+
+# Install per-repo binaries
+# Anything that is installed for use with go generate needs to be installed
+# here for correct operation on MacOS and Windows platforms.
+## https://github.com/istio/istio.io
```

Comment:
These are used by the istio/api repo.

Reply:
I don't think it's just about mac/windows. It's about having the build tools in the build container image in general. That's the point of the container image.

Reply:
will address.